includes/class-img-converter.php

```
@@ -462,6 +470,133 @@ private function convert_palette_to_truecolor( $image ) {
+			$data    = ob_get_clean();
+
+			// phpcs:ignore
+			imagedestroy( $thumb );
+
+			if ( ! $success || false === $data ) {
+				return '';
+			}
+
+			// phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_encode
+			return 'data:image/jpeg;base64,' . base64_encode( $data );
+		}
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 556. (undefined)

(ErrorControlOperator)